modules/core/src/test/java/org/locationtech/jts/math/DDIOTest.java

```diff
@@ -88,6 +88,11 @@ private void checkSciNotation(DD x, String expectedStr) {
 	}
 
 	public void testParse() {
+		//checkParse("0", 0, 1e-32);
+		checkParse("1", 1, 1e-32);
+        checkParse("100", 100, 1e-32);
```

Comment:
Would be nice to keep indentation consistent

Reply:
Agreed